### ISSUE TITLE
Hi my friend, need your help!

### ISSUE THREAD

**Issue by huHHhhuhu** [petewarden/dstk#68]
Hi my friend,

Sorry for asking an irrelevant question here,
I read your comment here:
tensorflow/tensorflow#13868
I'm doing the same project as you were, and could you help tell me the output node name of the low latency svdf? Because there are too many options even if I use the summary_graph tool:

Found 19 possible outputs: (name=data/Merge/MergeSummary, op=MergeSummary) (name=runtime-memory/read, op=Identity) (name=cross_entropy/sparse_softmax_cross_entropy_loss/xentropy/Shape, op=Shape) (name=cross_entropy/sparse_softmax_cross_entropy_loss/assert_broadcastable/values/shape, op=Shape) (name=cross_entropy/sparse_softmax_cross_entropy_loss/num_present/broadcast_weights/assert_broadcastable/values/shape, op=Shape) (name=train/gradients/cross_entropy/sparse_softmax_cross_entropy_loss/value_grad/tuple/control_dependency_1, op=Identity) (name=train/gradients/cross_entropy/sparse_softmax_cross_entropy_loss/div_grad/tuple/control_dependency_1, op=Identity) (name=train/gradients/cross_entropy/sparse_softmax_cross_entropy_loss/Mul_grad/tuple/control_dependency_1, op=Identity) (name=train/gradients/zeros_like, op=ZerosLike) (name=train/gradients/dropout_2/mul_grad/tuple/control_dependency_1, op=Identity) (name=train/gradients/dropout_2/div_grad/tuple/control_dependency_1, op=Identity) (name=train/gradients/dropout_1/mul_grad/tuple/control_dependency_1, op=Identity) (name=train/gradients/dropout_1/div_grad/tuple/control_dependency_1, op=Identity) (name=train/gradients/dropout/mul_grad/tuple/control_dependency_1, op=Identity) (name=train/gradients/dropout/div_grad/tuple/control_dependency_1, op=Identity) (name=train/gradients/conv1d/Conv2D_grad/tuple/control_dependency, op=Identity) (name=confusion_matrix/SparseTensorDenseAdd, op=SparseTensorDenseAdd) (name=save/control_dependency, op=Identity) (name=Merge/MergeSummary, op=MergeSummary)

Thanks!
David
